generator_gomod.go

```diff
@@ -0,0 +1,102 @@
+package dalec
+
+import (
+	"github.com/moby/buildkit/client/llb"
+	"github.com/pkg/errors"
+)
+
+const (
+	gomodCacheDir  = "/go/pkg/mod"
+	gomodGenerator = "gomod"
```

Comment:
not sure if I missed it but is this const `gomodGenerator` used?

Reply:
Done.